test/functional/ui/syntax_keyword_item_spec.lua

```diff
@@ -0,0 +1,66 @@
+local helpers = require('test.functional.helpers')
+local Screen = require('test.functional.ui.screen')
+local clear = helpers.clear
+local execute = helpers.execute
+local insert = helpers.insert
+local eq = helpers.eq
+local eval = helpers.eval
+
+describe('Syntax', function()
+  local screen
+  local colors = Screen.colors
+
+  before_each(function()
+    clear()
+    screen = Screen.new()
```

Comment:
consider `Screen.new(25, 5)` or something like that for a smaller screen. See e.g. `mouse_spec.lua`. Unless you plan on needing the space for the tests, it's nice to condense the test files to make them easier to read.